cli/src/utils/accounts.rs

```diff
@@ -0,0 +1,112 @@
+use aze_lib::accounts::create_basic_aze_player_account;
+use aze_lib::client::{
+    self,
+    create_aze_client,
+    AzeAccountTemplate,
```

Comment:
You should not name the folders as `utils`. Better to the follow the conversion we followed we in node

like `account/mod.rs`

i'd also recommend you to explore `miden-client` repo structure

https://github.com/0xPolygonMiden/miden-client/tree/main

You can do the above changes and as a overall feedback on the organizational front you can do it like the way it's maintaing a `cli` module for all cli actions and `main.rs` one directory about `cli` just for initializing stuff.